### DISCUSSION TITLE
Rename mkdocs-tech-docs-template

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [ministryofjustice/mkdocs-tech-docs-template#42] [Polls]
The [tech-docs-template](https://github.com/alphagov/tdt-documentation) is likely to be deprecated so it makes less sense to call this project mkdocs-tech-docs-template. What do you think it should be renamed to?

**Comment by SoumayaMauthoorMOJ** [comment]
@michalc @hrahim-moj @jafacakes2011  would be good to get your opinion!

**Reply by CamLamb** [reply]
I've voted for `mkdocs-material-govuk` as I think the name being specific to the theme we are extending from makes the project easier to find. It also describes it better than the current name.

**Reply by michalc** [reply]
I went for mkdocs-govuk. I know it extends from mkdocs-material, but it leaves the route open to not, just in case…

**Comment by mamonu** [comment]
likewise i think `mkdocs-material-govuk` is the best option out of the ones in the poll.

**Reply by Crypto-Venkat** [reply]
Good

**Comment by hrahim-moj** [comment]
What about `govuk-mkdocs-material`? I like the idea of leading with govuk for clarity, and including `mkdocs-material` helps to clarify what the template is more than `govuk-mkdocs`. `govuk-mkdocs-template` could also be an option, depending on how important it is for users to know about the material aspect.

**Reply by SoumayaMauthoorMOJ** [reply]
I do like that - unfortunately I can't add it as an option without loosing all the existing votes. I'll assume that votes for "some other option" relates to this option :-)

**Reply by gwionap** [reply]
Had this been an option at the time I voted I would have voted for this as an option. Unfortunately it doesn't look like I can change my vote.

**Comment by SoumayaMauthoorMOJ** [comment]
It turns out that [eleventy](https://www.11ty.dev/docs/plugins/) also has the convention eleventy-{plugin}, but the team behind [govuk-eleventy-plugin](https://github.com/x-govuk/govuk-eleventy-plugin) thought it was more important to start with govuk first to highlight that it pertains to govuk (see https://github.com/x-govuk). I also like the fact that in terms of importance, govuk comes first, then mkdocs, and finally material. Hence we've decided to go ahead with `govuk-mkdocs-material`, unless someone points out an issue with this suggestion.
